### ISSUE TITLE
more regtests are needed

### ISSUE THREAD

**Issue by carlocamilloni** [multi-ego/multi-eGO#317]
In particular I think that:

- [x] lysozyme + benzene can cover the cases of multi-components systems and of multi-domain proteins (this is on hold waiting for #332)
- [x] popc can cover the case of user provided pairs (this can be added only after #306) 
- [x] in cmdata we need to use lyz+benzene to cover all possibly type of histograms (this is on hold waiting for #332)
- [x] domain_sectioner is also not covered yet
- [ ] we need popc also for mego with user pairs
